### DISCUSSION TITLE
Мониторинг состояния детектора движения на домофоне

### DISCUSSION THREAD

**Discussion by cruncher1982** [rosteleset/SmartYard-Server#70] [Ideas]
Мониторинг состояния детектора движения через обработку событий motionDetection.
При эксплуатации столкнулись с тем что домофон был полностью функционален кроме датчика движения.Связываем это с выполнением работ перед камерой домофона на протяжении пару часов.Пепезагрузка домофона оживила датчик движения.Такое поведение было на всех трёх домофонах где проводили отделочные работы.

**Comment by subs1stem** [comment]
Вы предлагаете перезагружать домофоны автоматически в таком случае? А выключить/включить детектор пробовали?

**Reply by cruncher1982** [reply]
Пробовал,не помогало.Я думаю достаточно что забиксе будет трубить,что в предыдущие полчаса не было событий motionDetection.А там уже человек будет смотреть и проверять.

**Reply by subs1stem** [reply]
Ночью детектор движения вполне может молчать по естественным причинам до 5-7 часов подряд. Да и днём полчаса отсутствия движения очень частая ситуация, особенно при правильно настроенной зоне детекции (если она настраивается). Как отличить? Пока выглядит как баг, который должен исправлять вендор)

**Reply by cruncher1982** [reply]
Важно чтоб днём всё алерты закрылись,а если остались,то сразу можно глянуть конкретный домофон.В забиксе и интервалы настраиваются в тригере за какое время считать уже событие алертом.
